### PR TITLE
fix(dashboard): persist language choice

### LINKED ISSUE #774: Preferred language is not saved
On the dashboard, switching the preferred language isn't saved. When accessing again the dashboard, it will not use the selected language and fallback to `en`.

We should save it and load it from `localStorage`.

### PR DESCRIPTION
**Pull request by diboune**
## About

Fixes #774

### CODE REVIEW

File: packages/dashboard/locales/index.ts [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+  };
+};
+
+export const usePersistLocale = () => {
+  const changeLocale = useChangeLocale();
+
+  useEffect(() => {
+    const locale = localStorage.getItem('locale') as Parameters<typeof changeLocale>[0] | null;
+    if (locale) {
+      console.log(locale);
```
Comment (QuiiBz) [inline packages/dashboard/locales/index.ts:29]: Could you remove this `console.log`?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks for this PR! I only have one comment:

File: packages/dashboard/locales/index.ts (hunk 1 above)
Comment (diboune) [reply packages/dashboard/locales/index.ts:29]: missed it, will do

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Forgot to mention it in the initial review, but could you run `pnpm changeset` at the root for `@lagon/dashboard` as a `patch`? See https://docs.lagon.app/contributing#before-submitting-a-pr

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**